src/Database/Driver/Sqlserver.php

```diff
@@ -192,7 +192,10 @@ public function enabled(): bool
     public function prepare($query): StatementInterface
     {
         $this->connect();
-        $options = [PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL];
+        $options = [
+            PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL,
+            PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE => PDO::SQLSRV_CURSOR_BUFFERED,
+        ];
```

Comment:
Should we allow connection/driver level configuration to choose the cursor type? If users are loading large results they might not want client side cursors.

Reply:
`Query::disableBufferedResults()` will clear the cursor setting (just below this code). 

Do we have a global way to set that for other dbs? I don't think postgres has any buffering settings. Mysql uses the same `Query::disableBufferedResults()` flag.

Reply:
I wonder if any users need options like PDO::SQLSRV_CURSOR_STATIC for server-side cursors.

Reply:
I wouldn't personally use server-side cursors due to performance, but since they are an option available in the drivers, I believe any chance for users to _customize_ it are well welcomed.

Reply:
I think we can solve this through a Configure setting that applies defaults globally instead of per-call.